### PR TITLE
Add table anchors in About billing for GitHub Actions

### LINKED ISSUE #8368: Add table anchors in About billing for GitHub Actions
### What article on docs.github.com is affected?

https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions

### What part(s) of the article would you like to see updated?

Each of these should have a section so that one could easily link to it:
https://github.com/github/docs/blob/77e4e47c7460acbc0ff6a41897fd2854cc1bf5e3/content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md#L26-L32
https://github.com/github/docs/blob/77e4e47c7460acbc0ff6a41897fd2854cc1bf5e3/content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md#L36-L40
https://github.com/github/docs/blob/77e4e47c7460acbc0ff6a41897fd2854cc1bf5e3/content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md#L52-L56
https://github.com/github/docs/blob/77e4e47c7460acbc0ff6a41897fd2854cc1bf5e3/content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md#L64-L68
https://github.com/github/docs/blob/77e4e47c7460acbc0ff6a41897fd2854cc1bf5e3/content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md#L70-L75

### Additional information

I want to be able to link to the section that identifies macos runners as considerably more expensive than linux runners, but I can't because there are no sections for the tables.

### PR DESCRIPTION
**Pull request by jsoref**
### Why:

closes #8368

### What's being changed:

Adding anchors

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@jsoref Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by ivan0117 — APPROVED** [review]
p

**Review by ivan0117 — APPROVED** [review]
p

**Comment by ivan0117** [comment]
jsoref:billing-anchors

**Comment by ivan0117** [comment]
https://github.com/github/docs/pull/8390#issue-693589837

**Review by SalDenizardJr — DISMISSED** [review]

File: content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ If you purchased {% data variables.product.prodname_enterprise %} through a Micr
 
 Minutes reset every month, while storage usage does not.
 
+### Included Storage and Minutes per month
```
Comment (lecoursen) [inline content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md:26]:
```suggestion
### Included storage and minutes
```

File: content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md [hunk 2]
```diff
@@ -33,6 +35,8 @@ Minutes reset every month, while storage usage does not.
 
 Jobs that run on Windows and macOS runners that {% data variables.product.prodname_dotcom %} hosts consume minutes at 2 and 10 times the rate that jobs on Linux runners consume. For example, using 1,000 Windows minutes would consume 2,000 of the minutes included in your account. Using 1,000 macOS minutes, would consume 10,000 minutes included in your account.
 
+### Operating system minute multiplier
```
Comment (lecoursen) [inline content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md:38]:
```suggestion
### Minute multipliers
```

File: content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md [hunk 3]
```diff
@@ -49,6 +53,8 @@ The storage used by a repository is the total storage used by {% data variables.
 
 {% endnote %}
 
+### Operating system Per-minute rate
```
Comment (lecoursen) [inline content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md:56]:
```suggestion
### Per-minute rates
```

**Review by lecoursen — COMMENTED** [review]
Thanks again @jsoref! I'll apply a few tweaks and merge. This is going to really help folks navigate this article. ✨

**Review by lecoursen — DISMISSED** [review]

**Review by lecoursen — APPROVED** [review]

**Pull request merged**